server/server.js

```diff
@@ -15,9 +14,36 @@ const androidVariantCRD = require('./android-variant-crd.json');
 const iosVariantCRD = require('./ios-variant-crd.json');
 const webPushVariantCRD = require('./web-variant-crd.json');
 const mobileSecurityService = require('./mobile-security-crd.json');
+const proxy = require('http-proxy-middleware');
+
+const OPENSHIFT_PROXY_PATH = '/proxy/openshift';
+
+process.on('uncaughtException', error => {
+  console.log('uncaughtException', error);
+});
+
+process.on('unhandledRejection', error => {
+  console.log('unhandledRejection', error);
+});
 
 const app = express();
 
+const httpProxyOptions = {
+  target: addProtocolIfMissing(process.env.OPENSHIFT_HOST, 'https'),
+  secure: false,
+  logLevel: 'warn',
+  changeOrigin: true,
+  ws: true,
+  pathRewrite: {
+    [`^${OPENSHIFT_PROXY_PATH}`]: '' // remove base path
+  },
+  onProxyReq: (proxyReq, req) => {
+    proxyReq.setHeader('authorization', `Bearer ${req.headers['x-forwarded-access-token']}`);
```

Comment:
just thinking, does this work in dev environment? It might be no such header when there is no oauth-proxy running?